crates/subspace-farmer/src/single_disk_farm.rs

```diff
@@ -976,6 +976,45 @@ impl SingleDiskFarm {
             (Some(sender), Some(receiver))
         };
 
```

Comment:
`Benchmark random plotted sectors internally...` commit comment doesn't correspond to the commit code: the commit changes farmer code rather than benchmark code.